timetable/DB/DBConnect.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Data;
+using System.Data.SqlClient;
+using System.Configuration;
+using System.Data.Entity;
+
+namespace Timetable.timetable.DB
+{
+    public class DBConnect : System.Data.Entity.DbContext
+    {
+ 
+        public DBConnect() : base("name = DataModel")
+        {
+                
+        }
+
+        public void Connect(){
```

Comment:
Unused method.

Reply:
huh, i do not have this class hahah

Reply:
Found it, it was hidden. Removed the class